CHANGELOG.md

```diff
@@ -10,6 +10,64 @@ To see unreleased changes, please see the [CHANGELOG on the main branch guide](h
 
 <!-- towncrier release notes start -->
 
+## [0.18.0] - 2023-01-17
+
+### Packaging
+
+- Relax `indexmap` optional depecency to allow `>= 1.6, < 2`. [#2849](https://github.com/PyO3/pyo3/pull/2849
+- Relax `hashbrown` optional dependency to allow `>= 0.9, < 0.14`. [#2875](https://github.com/PyO3/pyo3/pull/2875)
+- Update `memoffset` dependency to 0.8. [#2875](https://github.com/PyO3/pyo3/pull/2875)
+
+### Added
+
+- Add `GILOnceCell::get_or_try_init` for fallible `GILOnceCell` initialization. [#2398](https://github.com/PyO3/pyo3/pull/2398)
+- Added experimental feature `experimental-inspect` with `type_input()` and `type_output()` helpers to get the Python type of any Python-compatible object. [#2490](https://github.com/PyO3/pyo3/pull/2490) [#2882](https://github.com/PyO3/pyo3/pull/2882)
```

Comment:
Some of them use `Add` and others use `Added`, `Add` was frequently used in previous changelog entries, I think we should use it consistently.

---

BTW, 0.16.6 changelog seems to have a minor markdown syntax issue: https://github.com/PyO3/pyo3/blob/71f06f916ef4140b7deef0ebf99efa902e413717/CHANGELOG.md#0166---2022-08-23

<img width="384" alt="image" src="https://user-images.githubusercontent.com/1556054/212847395-15f0f12c-2e95-4413-ae6f-502e933d654a.png">

Reply:
Good spot, will change to `Add`.

The markdown syntax issue is an nbsp - I keep accidentally hitting those by holding option key too long when hitting space. Got a plugin in my editor to highlight them now to avoid adding more. Will fix this one. 🤦